apps/meteor/client/views/omnichannel/currentChats/FilterByText.tsx

```diff
@@ -5,7 +5,7 @@ import moment from 'moment';
 import React, { Dispatch, FC, SetStateAction, useEffect } from 'react';
 
 import AutoCompleteAgent from '../../../components/AutoCompleteAgent';
-import AutoCompleteDepartment from '../../../components/AutoCompleteDepartment';
+import { AutoCompleteDepartment } from '../../../components/AutoCompleteDepartment';
```

Comment:
why are we exporting a `memo(AutoCompleteDepartment)` and using the named export?

Reply:
We have a pattern to use named export instead of default export, right? However, on `apps/meteor/ee/client/omnichannel/additionalForms/register.ts` we have a dynamic import, forcing me to sustain the default export too...

Reply:
> We have a pattern to use named export instead of default export, right?

Wrong. https://developer.rocket.chat/fuselage/conventions#react

> Most of time we prefer using named exports, but [it] provides a more readable code when using HOCs like `memo` and `forwardRef` and it ties nicely to `lazy`.

Reply:
believe me just import from the default (and memoized)

Reply:
hmmmmmmm, ok... updating until the EOD...

Reply:
sorry if I misunderstood how we deal with our code regarding our definitions. 😬
Thanks for your help on it ☺️